src/ui/zcl_abapgit_gui_obj_filter_tr.clas.abap

```diff
@@ -0,0 +1,65 @@
+CLASS zcl_abapgit_gui_obj_filter_tr DEFINITION
```

Comment:
Let's move this popup to `zcl_abapgit_popups`. Ideally, it should be able to replace the same popup in `zcl_abapgit_transport_mass / select_tr_requests` so we will have only one central place asking for multiple transport requests.

Reply:
Ok I will move it there and add some parameters to make it flexible.

Reply:
I added the methods popup_select_tr_requests and popup_select_wb_tc_tr_and_tsk to the interface zif_abapgit_popups (class zcl_abapgit_popups).
Popup_select_tr_requests selects the transports with imported parameters. Popup_select_wb_tc_tr_and_tsk calls popup_select_tr_requests with the parameters to select workbench and transport of copies transports and tasks.

So the method zcl_abapgit_transport_mass / select_tr_requests can be adjusted to call one of this methods later.